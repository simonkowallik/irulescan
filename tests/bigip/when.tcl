when HTTP_REQUST {
    expr 1
}
when HTTP_REQUST priority 2 {
    expr 2
}
when HTTP_REQUST timing on {
    expr 3
}
when HTTP_REQUST timing on priority 4 {
    expr 4
}