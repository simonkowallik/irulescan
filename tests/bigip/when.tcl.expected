WARNING: Unquoted expr at `1` in `expr 1`
WARNING: Unquoted expr at `2` in `expr 2`
WARNING: Unquoted expr at `3` in `expr 3`
WARNING: Unquoted expr at `4` in `expr 4`

